ecdsa_test.go

```diff
@@ -155,10 +157,24 @@ func BenchmarkECDSASigning(b *testing.B) {
 				if err != nil {
 					b.Fatalf("[%v] Error signing token: %v", data.name, err)
 				}
-				if sig == parts[2] {
+				if reflect.DeepEqual(sig, decodeSegment(b, parts[2])) {
```

Comment:
This feels more like a test, do we need to assert here?

Reply:
I am not quite sure what you mean

Reply:
Ah, because it’s a benchmark you mean? Good point